.github/workflows/ci.yml

```diff
@@ -34,3 +34,5 @@ jobs:
         run: rebar3 test
       - shell: bash
         run: rebar3 escriptize && _build/default/bin/elvis rock
+      - shell: bash
+        run: rebar3 as test test
```

Comment:
I think this effectively replaces the meta-checks, but lemme know.

Reply:
It does.

Reply:
Do remove the meta suite.

Reply:
It's removed already.